app/grandchallenge/workstations/templatetags/workstations.py

```diff
@@ -79,6 +79,9 @@ def get_workstation_path_and_query_string(  # noqa: C901
 def workstation_session_control_data(
     *, workstation, context_object, timeout=False, **kwargs
 ):
+    if workstation is None:
+        return ""
```

Comment:
Seriously? So just keep the workstation button broken unless a workstation is defined?

Reply:
that is a silly solution, my bad. I made a new commit where the button only shows if the workstation is not none, and tested it. It looks like this now when there is no workstation.

![Screenshot 2024-09-24 at 11 54 25](https://github.com/user-attachments/assets/8b0b2433-b5df-4796-b3dd-e6b46cd55a99)

Reply:
Really? Yet again, not a solution. The user needs to be able to view the archive item.

Reply:
It looks like I don't understand what is a workstation and how does it relate to an archive item.
To my understanding, a workstation is the medical image viewer (in this case, Cirrus). Then, the "view in viewer" button should open the medical image linked to the archive item (as CIV) in the assigned workstation to the archive.
If there is no workstation assigned to the archive, that means any image linked to any archive item can not be viewed using "view in viewer" button. However, the archive item details view should still be accessible (as shown in the screenshot above) but without showing the "view in viewer" button.

Can you please give me feedback on how I understand it?

Reply:
Specifying a workstation on any object just means use _this_ workstation. Otherwise, use the default.